tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java

```diff
@@ -309,6 +315,31 @@ public static void openAsProject( SDK sdk )
                 }
 
                 sdkProject.open( npm );
+
+                IFilterMatcherDescriptor descriptor = null;
+                IFilterMatcherDescriptor[] descriptors = ResourcesPlugin.getWorkspace().getFilterMatcherDescriptors();
+
+                for( int i = 0; i < descriptors.length; i++ )
+                {
+                    if( descriptors[i].getName().equals( "AND" ) )
+                    {
+                        descriptor = descriptors[i];
+                    }
+                }
+
+                if( descriptor != null )
+                {
+                    FileInfoMatcherDescription fmd = new FileInfoMatcherDescription( descriptor.getId(), null );
+
+                    IFolder[] folders = { sdkProject.getFolder( "portlets" ), sdkProject.getFolder( "layouttpl" ),
```

Comment:
Does this filter the portlets, hooks the entire folders themselves or does it filter the children of those folders but leave the actual folders?